README.md

```diff
@@ -93,21 +79,77 @@ versions of these dependencies:
 | gRPC    | v1.10.x |
 | libcurl | 7.47.0  |
 
-For Linux, the `ci/Dockerfile.*` files are a good reference on how to install
-all the necessary dependencies on each distribution. For Windows,
-consult [ci/install-windows.ps1](ci/install-windows.ps1).
+#### Tests
 
-On macOS, the following commands should install all the dependencies you need:
+Integration tests at times use the
+[Google Cloud SDK](https://cloud.google.com/sdk/). The integration tests run
+against the latest version of the SDK on each commit and PR.
+
+## Install Dependencies
+
+#### Fedora
+
+```bash
+dnf makecache
```

Comment:
Maybe we should create a script like `install-macosx.sh` and `install-windows.ps1` that figures out what distribution is running and executes the right commands?

Reply:
That would be helpful for linux distros but may be too much?